File: 8KYU/Sum of differences in array/solution.js
function sumOfDifferences(arr) {
    if(arr.length < 2)
      return 0;
      
    var sorted = arr.sort((a,b) => b-a);
    var answer = 0;
    for(var i=0; i < sorted.length-1; i++)
      answer += (sorted[i] - sorted[i+1]);
    return answer;
}

File: 8KYU/Sum of differences in array/solution.cs
using System;
using System.Linq;
public static class Kata
{
  public static int SumOfDifferences(int[] arr)
  {
     if(arr.Length < 2) return 0;
     var sorted = arr.OrderByDescending(x => x).ToList();
     var answer = 0;
     for(var i = 0; i < sorted.Count()-1; i++)
     {
       answer += (sorted[i] - sorted[i+1]);
     }
     return answer;
  }
}